src/log/access_log.cc

```diff
@@ -73,9 +73,27 @@ static void fvdbRegisterWithCacheManager();
 
 int LogfileStatus = LOG_DISABLE;
 
+// TODO: resolve duplication with Format.cc
```

Comment:
Please keep SQUID-1026-chrono-to-legacy (PR 1915) in mind when addressing this TODO. No rush!

Reply:
Ok, I'll apply that PR when it merges.